tools/pd-ctl/tests/config/config_test.go

```diff
@@ -343,6 +343,48 @@ func (suite *configTestSuite) checkConfig(cluster *pdTests.TestCluster) {
 	re.Contains(string(output), "is invalid")
 }
 
+func (suite *configTestSuite) TestConfigForwardControl() {
+	re := suite.Require()
+	re.NoError(failpoint.Enable("github.com/tikv/pd/pkg/dashboard/adapter/skipDashboardLoop", `return(true)`))
+	suite.env.RunTestInTwoModes(suite.checkConfigForwardControl)
+	re.NoError(failpoint.Disable("github.com/tikv/pd/pkg/dashboard/adapter/skipDashboardLoop"))
+}
+
+func (suite *configTestSuite) checkConfigForwardControl(cluster *pdTests.TestCluster) {
+	re := suite.Require()
+	leaderServer := cluster.GetLeaderServer()
+	pdAddr := leaderServer.GetAddr()
+	cmd := ctl.GetRootCmd()
+
+	args := []string{"-u", pdAddr, "config", "show", "--from_api_server"}
```

Comment:
How do we know if this flag is effective?

Reply:
I have added more tests to check